Library/Formula/launchdns.rb

```diff
@@ -15,12 +15,14 @@ class Launchdns < Formula
 
   def install
     ENV["PREFIX"] = prefix
+    system "./configure", "--with-launch-h", "--with-launch-h-activate-socket"
     system "make", "install"
 
     (prefix+"etc/resolver/dev").write("nameserver 127.0.0.1\nport 55353\n")
   end
 
   test do
+    assert_not_match /without socket activation/, shell_output("#{bin}/launchdns --version")
```

Comment:
Unfortunately `assert_not_match` is not available in Ruby 1.8. You can use `assert_no_match` instead.